docker_test.go

```diff
@@ -115,17 +106,13 @@ func TestContainerWithHostNetworkOptions_UseExposePortsFromImageConfigs(t *testi
 	require.NoError(t, err)
 
 	endpoint, err := nginxC.Endpoint(ctx, "http")
-	if err != nil {
-		t.Errorf("Expected server endpoint. Got '%v'.", err)
-	}
+	require.NoErrorf(t, err, "Expected server endpoint")
 
 	resp, err := http.Get(endpoint)
 	require.NoError(t, err)
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
-		t.Errorf("Expected status code %d. Got %d.", http.StatusOK, resp.StatusCode)
-	}
+	assert.Equalf(t, http.StatusOK, resp.StatusCode, "Expected status code %d. Got %d.", http.StatusOK, resp.StatusCode)
```

Comment:
suggestion: use require not assert, more below

Reply:
That would just cause churn if tests are updated as well confusion, so lets keep it simple and use require everywhere please.